my-project/public/js/ProductCategoryScripts/Shared.js

```diff
@@ -24,11 +30,30 @@ function RemoveImageFromCategory(path, img)
         })
         .then((response)=>{
             img.remove();
-            alert(response.message);
+            BootstrapAlert("success", response.message);
         })
         .catch((err)=>{
             err.then((e)=>{
-                alert(e.message);
+                BootstrapAlert("danger", e.error);
             });
         });
+}
+
+/**
+ * @param {string} typeOfMessage
+ * @param {string} message
+ * @returns void
+ * */
+function BootstrapAlert(typeOfMessage, message){
```

Comment:
@mSobonVirtua better to have message as a first param, than type and if type not set just default to error for example